client/components/jetpack/jetpack-disconnected/index.tsx

```diff
@@ -24,22 +23,18 @@ const JetpackDisconnected: FunctionComponent = () => {
 	const reconnectUrl = `https://wordpress.com/settings/disconnect-site/${ siteSlug }?type=down`;
 	const body = [
 		<span className="jetpack-disconnected__paragraph" key="paragraph-1">
-			{ preventWidows(
-				translate( 'Jetpack is unable to reach your site {{siteName/}} at this moment.', {
-					components: { siteName: <strong>{ siteName }</strong> },
-				} )
-			) }
+			{ translate( 'Jetpack is unable to reach your site {{siteName/}} at this moment.', {
+				components: { siteName: <strong>{ siteName }</strong> },
+			} ) }
 		</span>,
 		<span className="jetpack-disconnected__paragraph" key="paragraph-2">
-			{ preventWidows(
-				translate(
-					'Please visit {{siteUrl/}} to ensure your site is loading correctly and reconnect Jetpack if necessary.',
-					{
-						components: {
-							siteUrl: <ExternalLink href={ siteUrl }>{ siteUrl }</ExternalLink>,
-						},
-					}
-				)
```

Comment:
Not sure how to test this but `bodyText` in the Upsell component is rendered inside a `p`, so this should be OK.